modules/apache-airflow/templates/values.yaml

```diff
@@ -2105,6 +2103,7 @@ config:
     # For Airflow 1.10, backward compatibility; moved to [logging] in 2.0
     colored_console_log: 'False'
     remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
+    allowed_deserialization_classes: "airflow.* DownloadMetric SnapshotMetrics LENSDataset Dataset Syn.*"
```

Comment:
Nice work here, is this a risk if we have to keep adding to this list?

Reply:
I think having to specify this is a ridiculous requirement that they added. I am trying to see if I can just wild card allow anything. Then if it cannot be de serialized, then fail at runtime.